config.json

```diff
@@ -1457,57 +1617,99 @@
       {
         "groupId": "androidx.wear.compose",
         "artifactId": "compose-foundation",
-        "version": "1.1.2",
-        "nugetVersion": "1.1.2.3",
+        "version": "1.2.0",
+        "nugetVersion": "1.2.0",
         "nugetId": "Xamarin.AndroidX.Wear.Compose.Foundation",
         "dependencyOnly": false
       },
       {
         "groupId": "androidx.wear.compose",
         "artifactId": "compose-material",
-        "version": "1.1.2",
-        "nugetVersion": "1.1.2.3",
+        "version": "1.2.0",
+        "nugetVersion": "1.2.0",
         "nugetId": "Xamarin.AndroidX.Wear.Compose.Material",
         "dependencyOnly": false
       },
+      {
+        "groupId": "androidx.wear.compose",
+        "artifactId": "compose-material-core",
+        "version": "1.2.0",
+        "nugetVersion": "1.2.0",
+        "nugetId": "Xamarin.AndroidX.Wear.Compose.Material.Core",
+        "dependencyOnly": false
+      },
       {
         "groupId": "androidx.wear.compose",
         "artifactId": "compose-navigation",
-        "version": "1.1.2",
-        "nugetVersion": "1.1.2.3",
+        "version": "1.2.0",
+        "nugetVersion": "1.2.0",
         "nugetId": "Xamarin.AndroidX.Wear.Compose.Navigation",
         "dependencyOnly": false
       },
+      {
+        "groupId": "androidx.wear.protolayout",
+        "artifactId": "protolayout",
+        "version": "1.0.0",
+        "nugetVersion": "1.0.0",
+        "nugetId": "Xamarin.AndroidX.ProtoLayout",
+        "dependencyOnly": false
+      },
+      {
+        "groupId": "androidx.wear.protolayout",
+        "artifactId": "protolayout-expression",
+        "version": "1.0.0",
+        "nugetVersion": "1.0.0",
+        "nugetId": "Xamarin.AndroidX.ProtoLayout.Expression",
+        "excludedRuntimeDependencies": "org.jetbrains.kotlin.kotlin-stdlib",
```

Comment:
Why exclude `kotlin-stdlib`? We bind that package.

Reply:
I think copypasta.

Reply:
No it was not copypasta without that line binderator crashes:

```
Unhandled exception. System.ArgumentNullException: Value cannot be null. (Parameter 'input')
   at System.Text.RegularExpressions.ThrowHelper.ThrowArgumentNullException(ExceptionArgument arg)
   at MavenNet.MavenVersionRange.Satisfies(String version)
   at MavenNet.Models.Dependency.Satisfies(String version)
   at AndroidBinderator.Engine.<>c__DisplayClass7_0.<BuildProjectModels>b__0(MavenArtifactConfig ma) in C:\a\_work\1\s\util\Xamarin.AndroidBinderator\Xamarin.AndroidBinderator\Engine.cs:line 333
   at System.Linq.Enumerable.TryGetFirst[TSource](IEnumerable`1 source, Func`2 predicate, Boolean& found)
   at AndroidBinderator.Engine.BuildProjectModels(BindingConfig config, Dictionary`2 mavenProjects) in C:\a\_work\1\s\util\Xamarin.AndroidBinderator\Xamarin.AndroidBinderator\Engine.cs:line 330
   at AndroidBinderator.Engine.ProcessConfig(BindingConfig config) in C:\a\_work\1\s\util\Xamarin.AndroidBinderator\Xamarin.AndroidBinderator\Engine.cs:line 75
   at AndroidBinderator.Engine.BinderateAsync(BindingConfig config) in C:\a\_work\1\s\util\Xamarin.AndroidBinderator\Xamarin.AndroidBinderator\Engine.cs:line 46
   at Xamarin.AndroidBinderator.Tool.Program.Main(String[] args) in C:\a\_work\1\s\util\Xamarin.AndroidBinderator\Xamarin.AndroidBinderator.Tool\Program.cs:line 67
   at Xamarin.AndroidBinderator.Tool.Program.<Main>(String[] args)
```

pom.xml

```
<?xml version="1.0" encoding="UTF-8"?>
<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
  <!-- This module was also published with a richer model, Gradle metadata,  -->
  <!-- which should be used instead. Do not delete the following line which  -->
  <!-- is to indicate to Gradle or any Gradle module metadata file consumer  -->
  <!-- that they should prefer consuming it instead. -->
  <!-- do_not_remove: published-with-gradle-metadata -->
  <modelVersion>4.0.0</modelVersion>
  <groupId>androidx.wear.protolayout</groupId>
  <artifactId>protolayout-expression</artifactId>
  <version>1.0.0</version>
  <packaging>aar</packaging>
  <name>ProtoLayout Expression</name>
  <description>Create dynamic expressions (for late evaluation by a remote evaluator).</description>
  <url>https://developer.android.com/jetpack/androidx/releases/wear-protolayout#1.0.0</url>
  <inceptionYear>2022</inceptionYear>
  <licenses>
    <license>
      <name>The Apache Software License, Version 2.0</name>
      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
      <distribution>repo</distribution>
    </license>
  </licenses>
  <developers>
    <developer>
      <name>The Android Open Source Project</name>
    </developer>
  </developers>
  <scm>
    <connection>scm:git:https://android.googlesource.com/platform/frameworks/support</connection>
    <url>https://cs.android.com/androidx/platform/frameworks/support</url>
  </scm>
  <dependencyManagement>
    <dependencies>
      <dependency>
        <groupId>androidx.wear.protolayout</groupId>
        <artifactId>protolayout</artifactId>
        <version>1.0.0</version>
      </dependency>
      <dependency>
        <groupId>androidx.wear.protolayout</groupId>
        <artifactId>protolayout-expression-pipeline</artifactId>
        <version>1.0.0</version>
      </dependency>
      <dependency>
        <groupId>androidx.wear.protolayout</groupId>
        <artifactId>protolayout-material</artifactId>
        <version>1.0.0</version>
      </dependency>
      <dependency>
        <groupId>androidx.wear.protolayout</groupId>
        <artifactId>protolayout-proto</artifactId>
        <version>1.0.0</version>
      </dependency>
      <dependency>
        <groupId>androidx.wear.protolayout</groupId>
        <artifactId>protolayout-renderer</artifactId>
        <version>1.0.0</version>
      </dependency>
    </dependencies>
  </dependencyManagement>
  <dependencies>
    <dependency>
      <groupId>androidx.annotation</groupId>
      <artifactId>annotation</artifactId>
      <version>1.2.0</version>
      <scope>compile</scope>
    </dependency>
    <dependency>
      <groupId>androidx.annotation</groupId>
      <artifactId>annotation-experimental</artifactId>
      <version>1.3.0</version>
      <scope>runtime</scope>
      <type>aar</type>
    </dependency>
    <dependency>
      <groupId>androidx.collection</groupId>
      <artifactId>collection</artifactId>
      <version>1.2.0</version>
      <scope>runtime</scope>
    </dependency>
    <dependency>
      <groupId>androidx.wear.protolayout</groupId>
      <artifactId>protolayout-proto</artifactId>
      <version>[1.0.0]</version>
      <scope>runtime</scope>
    </dependency>
    <dependency>
      <groupId>org.jetbrains.kotlin</groupId>
      <artifactId>kotlin-stdlib</artifactId>
      <scope>runtime</scope>
    </dependency>
  </dependencies>
</project>
```

Reply:
And there is also `Wear.` missing!!!!!

Reply:
`extraDependencies` crashes too.